<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UrlToXml</class>
 <widget class="QMainWindow" name="UrlToXml">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UrlToXml</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="statisticsTW">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>651</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(162, 162, 162);</string>
    </property>
    <property name="text">
     <string>将完整链接输入即可生成下方表单数据</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>输入链接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/create.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileBtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/checkfile.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #c1904d;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>数据表单</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>文件名</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>域名</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="xmlNameLineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>501</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(176, 176, 176);</string>
    </property>
    <property name="text">
     <string>编辑生成XML文件名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="domainNameLineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>501</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(176, 176, 176);</string>
    </property>
    <property name="text">
     <string>识别链接内域名</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="multiCBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>批量生成</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>参数名：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downLoadExcelBtn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>104</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>下载Excel模板</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importExcelBtn">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>104</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>导入Excel数据</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="fileTableWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>140</y>
      <width>311</width>
      <height>331</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>文件名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>参数值</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>描述</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="uniqueLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>未设置唯一识别值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>xml文件输出路径：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xmlPathLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseBtn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
